pkg/services/pluginsintegration/pluginstore/fake.go

```diff
@@ -12,6 +12,11 @@ type FakePluginStore struct {
 
 func (pr *FakePluginStore) Plugin(_ context.Context, pluginID string) (Plugin, bool) {
 	for _, v := range pr.PluginList {
+		for _, alias := range v.AliasIDs {
```

Comment:
Nit: Rather than duplicating logic here, maybe it makes sense to not mock the store + registry in the test in this case? Since we want to test the actual logic rather than "fake" logic